Styles/mobilemain.css

```diff
@@ -48,29 +52,19 @@
         content: "";
         display: block;
         width: 40px;
-        line-height: 5px;
-        margin-right: auto;
-        margin-left: auto;
         border-top: 5px solid #3A606E;
-        margin-top: 1em;
-        cursor: pointer;
     }
     .burger::after {
         position: absolute;
         content: "";
         display: block;
-        line-height: 5px;
         width: 40px;
-        margin-right: auto;
-        margin-left: auto;
         border-top: 5px solid #3A606E;
-        margin-top: 1em;
-        cursor: pointer;
     }
     .burger::after{
-        top:0px;
+        top:5px;
     }
     .burger::before{
-        top: -10px;
+        top: 15px;

```

Comment:
No marks lost here, but going down instead of up with your subsequent bars works better with the 1em top margin of the main div.  It gives the icon a bit more vertical space between it and the top of the page